apps/src/code-studio/initApp/project.js

```diff
@@ -826,7 +827,18 @@ var projects = module.exports = {
       pathName += '/' + action;
     }
     return pathName;
-  }
+  },
+
+  saveThumbnail(blob) {
```

Comment:
A method comment would be nice, esp. with a little help on what `blob` should be passed in here.